src/datasources/asset/data-sources/calibration-forecast/CalibrationForecastDataSource.ts

```diff
@@ -101,7 +101,8 @@ export class CalibrationForecastDataSource extends AssetDataSourceBase {
         return null;
     }
 
-    processResultsGroupedByTime(result: DataFrameDTO, timeGrouping: AssetCalibrationForecastKey) {
+    processResultsGroupedByTime(result: DataFrameDTO, timeGrouping: AssetCalibrationForecastKey, options: DataQueryRequest) {
+        const originalValuesMap = this.constructDataLinkValuesMap(result, options);
```

Comment:
As the method name is `constructDataLinkValuesMap`, maybe `dataLinkValuesMap` would be more suitable name.